src/lib/settings/SettingsFolder.js

```diff
@@ -88,18 +88,16 @@ class SettingsFolder extends Map {
 	 * Plucks out one or more attributes from either an object or a sequence of objects
 	 * @since 0.5.0
 	 * @param  {...string} paths The paths to take
-	 * @returns {Object<string,*>}
+	 * @returns {Array<any>}
 	 * @example
-	 * const { x, y } = message.guild.settings.pluck('x', 'y');
+	 * const [x, y] = message.guild.settings.pluck('x', 'y');
 	 * console.log(x, y);
 	 */
 	pluck(...paths) {
-		const object = {};
-		for (const path of paths) {
+		return paths.map(path => {
 			const value = this.get(path);
-			if (typeof value !== 'undefined') object[path] = value instanceof SettingsFolder ? value.toJSON() : value;
-		}
-		return object;
+			return typeof value !== 'undefined' ? value instanceof SettingsFolder ? value.toJSON() : value : null;
```

Comment:
Defaulting values that don't exist to null is a bad idea... If I pluck `myValueThatTotallyDoesntExistPerMySchema` I expect it to be undefined, not null... And thats what the old code did